openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-video.js

```diff
@@ -32,7 +32,10 @@ var Video = (function() {
 			cnts = {
 				video: true
 			};
-			promise = navigator.getDisplayMedia(cnts);
+			if (b.major > 80)
```

Comment:
could you please add `{}` to this if/else?

Reply:
sure, force-pushed with requested changes.